src/Kraken/Store.hs

```diff
@@ -122,7 +123,10 @@ lookupExecutionPlan store _dontChaseDependencies@True targets =
     mapM (lookupTarget store) targets
 lookupExecutionPlan store _dontChaseDependencies@False targets = do
     mapM_ (lookupTarget store) targets
-    return $ foldDependencies ((++), []) (graphWithPriorities store) targets (\ (name, node) -> [(name, node)])
+    let plan graph = foldDependencies ((++), []) graph targets (\ (name, node) -> [(name, node)])
+        with = plan (graphWithPriorities store)
+        without = plan (graphWithoutPriorities store)
+    return $ intersectBy ((==) `on` fst) with without
```

Comment:
I think this would be more understandable as a `filter`. E.g. ```filter ((`elem` without) . fst) with```. (And then have `with = map fst $ plan ...`.)